test/guppy.tests.js

```diff
@@ -400,6 +401,15 @@ describe('guppy', function () {
       });
     });
 
+    describe('pre-commit gulp options', function () {
+      it('passes options to gulp.src', function () {
```

Comment:
Can we rename this test to 'should pass options to gulp.src'?

And then can you add another spec for guppy.src that tests the same?